R/extract_features.R

```diff
@@ -547,12 +532,31 @@ extract_features <- function(examples,
                              output_file = NULL,
                              max_seq_length = 128L,
                              layer_indexes = -4:-1,
-                             use_one_hot_embeddings = FALSE,
                              batch_size = 2L,
                              features = c(
                                "output",
                                "attention"
-                             )) {
+                             ),
+                             verbose = FALSE) {
+  if (is.character(examples)) {
+    examples <- make_examples_simple(seq_list = examples)
+  } else if (inherits(examples, "InputExample_EF")) {
+    # As a courtesy, now allow a single InputExample_EF object to be passed in.
+    examples <- list(examples) # nocov
```

Comment:
Why nocov on this? Seems easy enough to cover...

Reply:
yeah, I mostly got tired of adding new `extract_features` tests. Each one adds a few seconds to testing time. But I'll go ahead and add *just one more*. :)